fendermint/app/settings/src/lib.rs

```diff
@@ -326,12 +326,16 @@ impl Settings {
                     .ignore_empty(true) // otherwise "" will be parsed as a list item
                     .try_parsing(true) // required for list separator
                     .list_separator(",") // need to list keys explicitly below otherwise it can't pase simple `String` type
+                    .with_list_parse_key("tracing.file.domain_filter")
+                    .with_list_parse_key("tracing.file.events_filter")
                     .with_list_parse_key("resolver.connection.external_addresses")
                     .with_list_parse_key("resolver.discovery.static_addresses")
                     .with_list_parse_key("resolver.membership.static_subnets")
                     .with_list_parse_key("eth.cors.allowed_origins")
                     .with_list_parse_key("eth.cors.allowed_methods")
-                    .with_list_parse_key("eth.cors.allowed_headers"),
+                    .with_list_parse_key("eth.cors.allowed_headers")
+                    .with_list_parse_key("eth.tracing.file.domain_filter")
+                    .with_list_parse_key("eth.tracing.file.events_filter"),
```

Comment:
Are these necessary for the Eth API process? I think it should just reuse the general tracing configuration, and if we want to enable/disable Eth API tracing, we should have an Eth domain we can disable through the `domain_filter`.

Reply:
Nice, that sounds better. So maybe `EthSettings` should have a subset of the tracing settings? as in:
```
pub domain_filter: Option<Vec<String>>,
pub events_filter: Option<Vec<String>>,
```

Reply:
I'm happy to make that change if sounds right... would prefer to do in a diff PR.